feat(events): enhance styling for events screen branch names

ZahraTee opened the pull request:
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | iPhone X, FWVGA slider |
| Bug fix?         | no      |
| New feature?     | yes...?     |
| Includes tests?  | no      |
| All Tests pass?  | yes     |
| Related ticket?  | #521        |

---

## Screenshots

<!--
  Replace the images in the table below with screenshots of your changes before
  and after. If this is not applicable (i.e. absolutely NO visual changes), feel
  free to delete this section.
-->

| Before   | After    |
| -------- | -------- |
|![screen shot 2018-01-14 at 01 11 08](https://user-images.githubusercontent.com/5565340/34911878-fea1e93c-f8ca-11e7-8032-de8e0fb081cb.png)|![screen shot 2018-01-15 at 23 59 35](https://user-images.githubusercontent.com/5565340/34965993-4c5be3ae-fa50-11e7-967b-568e089ebdbc.png)|
|![screen shot 2018-01-14 at 01 10 52](https://user-images.githubusercontent.com/5565340/34911885-0f78abce-f8cb-11e7-8de5-07e67d5a95e4.png)|![screen shot 2018-01-15 at 23 58 06](https://user-images.githubusercontent.com/5565340/34965986-3fc5b606-fa50-11e7-9701-e76882907796.png)|

## Description

Updated the styling for branch links in events to match the GitHub style.
~Also changed the styling for the deleted branch names as well to be red, as I felt the grey was a bit too washed out and didn't match the updated default style.~

<!-- DO NOT MODIFY BELOW THIS LINE -->
<!-- ----------------------------- -->
<!-- GITPOINT_PR -->

machour commented:
Thank you for taking care of this!

I'm not really sure about coloring in red though, as it looks kind of "aggressive" to the eye. Maybe a darker gray on a light gray background for the box would feel more confortable and appear as disabled right away.

Also, what do you think about changing the Monospace font we're using? I think something like [Roboto Mono Regular](https://fonts.google.com/specimen/Roboto+Mono) would look crisp rendered a little smaller than the rest of the text and with a bit more left & right padding for the box. 🤔

ZahraTee commented:
@machour 

I changed it to grey and changed the code font to Inconsolata which is pretty similar to Consolas which is used on GitHub web. Although having just pushed that is a fairly significant change in terms of screens throughout the app. 🤔

So, increasing the branch label padding is tricky because you can't control padding or margins of nested <Text> elements. I've been trying to get UserListItem to accept Views for the title prop but it's not really working out so far.

machour commented:
@ZahraTee this looks so nice! 🎉 🎉 

Had a look at the padding thing, and as you said, it's uncontrollable with styles. But I managed to trick it `&nbsp;`-style 😄  
This does the trick:

```diff
 const Datestamp = styled.Text`
@@ -214,7 +221,8 @@ class Events extends Component {
         ) {
           return (
             <LinkBranchDescription>
-              {userEvent.payload.ref}
+              {' '}{userEvent.payload.ref}{' '}
             </LinkBranchDescription>
           );
         } else if (userEvent.payload.ref_type === 'repository') {
       case 'ReleaseEvent':
```

resulting in this rendering which looks pretty decent to me, what do you think ?
<img width="376" alt="screen shot 2018-01-16 at 12 17 41 am" src="https://user-images.githubusercontent.com/304450/34965275-2805a4b0-fa53-11e7-8194-46ac5223be03.png">

You will also need to copy the ttf file to `android/app/src/main/assets/fonts/` for it to work on Android (tested it for you):
<img width="314" alt="screen shot 2018-01-16 at 12 31 47 am" src="https://user-images.githubusercontent.com/304450/34965494-aec7b820-fa54-11e7-94e8-deb5977ded61.png">

/cc @housseindjirdeh for cosmetic changes approval

ZahraTee commented:
@machour Just FYI I did add the android font but it didn't get added to the commit the first time round because of the .gitignore file added in #401. Probably worth watching out for.

machour commented:
erm, there might be a logic flaw in #401 as I ran `yarn run link` and the font weren't copied :/
don't hesitate to open a new issue for this problem!

machour commented on src/auth/screens/events.screen.js:
```diff
@@ -219,7 +219,8 @@ class Events extends Component {
             <LinkBranchDescription>
-              {userEvent.payload.ref}
+              {' '}
+              {userEvent.payload.ref}{' '}
```
Same thing should be done for `case 'PushEvent':` line 288

housseindjirdeh commented on src/config/fonts.js:
```diff
@@ -4,5 +4,5 @@ export const fonts = {
   fontPrimarySemiBold: { fontFamily: 'Nunito-SemiBold' },
   fontPrimaryBold: { fontFamily: 'Nunito-Bold' },
-  fontCode: { fontFamily: 'Menlo' },
+  fontCode: { fontFamily: 'Inconsolata-Regular' },
```
👍
